<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEnvironmentVariableDialog</class>
 <widget class="QDialog" name="UserEnvironmentVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="browseDirButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse Dir...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFileButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse File...</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Variable name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Variable value:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserEnvironmentVariableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserEnvironmentVariableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
